easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-foss-2018a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CmdCp'
+
+name = 'lpsolve'
+version = '5.5.2.5'
+
+homepage = 'https://sourceforge.net/projects/lpsolve/'
+description = "Mixed Integer Linear Programming (MILP) solver"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['lp_solve_%(version)s_source.tar.gz']
```

Comment:
Please add a checksum here too in the same way.

Reply:
@Serpens Checksum here is still missing?

Please include `SHA256` checksums for all new easyconfigs, we'll start enforcing this soon (see #5005)

Reply:
I added it